packages/management/tanzu-auth/package.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
+metadata:
+  name: tanzu-auth.tanzu.vmware.com.1.6.0
+  namespace: tanzu-auth
```

Comment:
note: (just putting my thoughts out there) i don't know how this `Package` gets consumed, let alone how this `namespace` gets set. for example, in TKGm, i would imagine this becomes `tkg-system`, but i don't know exactly how that would happen

Reply:
+1 documenting this for us!